addons/account_fleet/models/fleet_vehicle.py

```diff
@@ -17,7 +17,7 @@ def _compute_move_ids(self):
             return
 
         moves = self.env['account.move.line'].read_group(
-            domain=[('vehicle_id', 'in', self.ids), ('move_id.state', '!=', 'cancel')],
+            domain=[('vehicle_id', 'in', self.ids), ('move_id.state', '!=', 'cancel'), ('move_id.move_type', 'in', ('in_invoice', 'in_refund'))],
```

Comment:
I knew I should have written it like this :weary: 
```python
        moves = self.env['account.move.line'].read_group(
            domain=[
                ('vehicle_id', 'in', self.ids), 
                ('move_id.state', '!=', 'cancel'),
            ],
            fields=['vehicle_id', 'move_id:array_agg'],
            groupby=['vehicle_id'],
        )
```

Reply:
I wanted to do that, but thought it would make a larger diff so I left it inline :sweat_smile: 
But I'll change it :slightly_smiling_face:

Reply:
I don't mind a little bit more diff if it makes things more readable (but don't tell that to everyone :shushing_face:, some people see one excessive line of diff as a declaration of war)